audoma/drf/fields.py

```diff
@@ -144,3 +145,68 @@ def __init__(self, choices, **kwargs):
         else:
             self.original_choices = choices
         super().__init__(choices, **kwargs)
+
+
+class SerializerMethodField(ExampleMixin, fields.Field):
+    def _parse_field(self, field):
+        if field is None:
+            return None
+        elif isinstance(field, fields.Field):
+            return field
+        else:
+            raise ValueError(
+                f"Incorrect type of field, field \
+                    must be an instance of rest_framework.fields.Field.\
+                        Passed value: {field}"
+            )
+
+    def __init__(self, *args, **kwargs) -> None:
+        self.method_name = kwargs.pop("method_name", None)
```

Comment:
`method_name`, `field` and 'writable' should be in header with default values. Wouldn't be necessary to do pop on them and also it will be easier to see default values,
 I think we shouldn't use `is_` prefix in `is_writable` as `read_only` also doesn't have such prefix.